cmd/catalog.go

```diff
@@ -50,18 +62,61 @@ var catalogListCmd = &cobra.Command{
 	},
 }
 
+var catalogListServiceCmd = &cobra.Command{
+	Use:   "services",
+	Short: "Lists all the services from service catalog",
+	Long:  "Lists all the services from service catalog",
+	Example: `  # List all services
+  odo catalog list services
+
+ # Search for a supported services
+  odo catalog search services mysql
+	`,
+	Args: cobra.ExactArgs(0),
+	Run: func(cmd *cobra.Command, args []string) {
+		client := getOcClient()
+		catalogList, err := svc.ListCatalog(client)
+		checkError(err, "unable to list services because Service Catalog is not enabled in your cluster")
```

Comment:
our user will not understand openshift error (so we have to simplify it)
if there is network error, process will not reach till this stage :P

Reply:
Hmm so the occlient creation would fail is what you are saying right? So can there be any other error like permissions/authorisation for using service catalog issue ? 

If not please ignore this comment

Reply:
I think this error message is okay for this situation